src/main/java/com/crowdin/client/screenshots/ScreenshotsApi.java

```diff
@@ -127,10 +128,10 @@ public void replaceTags(Long projectId, Long screenshotId, ReplaceTagsRequest re
      * @param projectId    project identifier
      * @param screenshotId screenshot identifier
      * @param request      request object
-     * @return newly created tag
+     * @return newly created tags
      */
-    public ResponseObject<Tag> addTag(Long projectId, Long screenshotId, List<AddTagRequest> request) throws HttpException, HttpBadRequestException {
-        TagResponseObject tagResponseObject = this.httpClient.post(this.url + "/projects/" + projectId + "/screenshots/" + screenshotId + "/tags", request, new HttpRequestConfig(), TagResponseObject.class);
+    public ResponseObject<List<Tag>> addTags(Long projectId, Long screenshotId, List<AddTagRequest> request) throws HttpException, HttpBadRequestException {
```

Comment:
I suggest not renaming the method name to keep the backward compatibility and keep it consistent with the Docs where this method has an 'Add Tag' name.

Reply:
ok

Reply:
fixed